### PR TITLE
Added Altearn.xyz and BuildTheEarth.net

### PR DESCRIPTION
**Pull request by fantomitechno**
So, I added two website as always x)

> Proof it's working : 

Altearn.xyz 
without 
image
and with the privacy mode 
image
(This website was realy easy because with the website construction I can just see that page your looking with the window title)

BuildTheEarth.net
image
without 
image
and with Hide map position 
image
(I'm realy proud of the follow with the map)
image
and finaly with privacy mode 
image

### CODE REVIEW

File: websites/A/Altearn/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "id": "563749920683720709"
+    },
+    "url": "altearn.xyz",
+    "description": {
+      "en": "Altearn is a non-profit association which aims to develop and implement solutions and events for educational, scientific or awareness purposes.",
+      "fr": "Altearn est une association à but non-lucratif qui vise à déveloper et mettre en place des solutions et événements dans un but pédagogique, scientifique ou de sensibilisation."
+    },
+    "service": "Altearn",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/5BHWXzu.png",
```
Comment (Bas950) [inline websites/A/Altearn/dist/metadata.json:14]: Please get a better quality pic

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by fantomitechno** [comment]
I asked one to the owner

**Comment by fantomitechno** [comment]
And that's it

**Review by Bas950 — APPROVED** [review]

**Comment by ririxidev** [comment]
Domain is not 2 months old 🤷

**Pull request closed without merging**